spanner/api/SpannerTest/Tests.cs

```diff
@@ -96,23 +100,14 @@ public void Dispose()
             catch (RpcException ex) when (ex.Status.StatusCode == StatusCode.NotFound) { }
         }
 
-        public string ProjectId { get; private set; } =
-            Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        public string ProjectId { get; private set; } = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
         // Allow environment variables to override the default instance and database names.
-        public string InstanceId { get; private set; } =
-            Environment.GetEnvironmentVariable("TEST_SPANNER_INSTANCE") ?? "my-instance";
-        private static readonly string s_randomDatabaseName = "my-db-"
-            + TestUtil.RandomName();
-        private static readonly string s_randomToBeCancelledBackupName = "my-backup-"
-            + TestUtil.RandomName();
-        private static readonly string s_randomRestoredDatabaseName = "my-db-"
-            + TestUtil.RandomName();
-        public string DatabaseId =
-            Environment.GetEnvironmentVariable("TEST_SPANNER_DATABASE") ?? s_randomDatabaseName;
+        public string InstanceId { get; private set; } = Environment.GetEnvironmentVariable("TEST_SPANNER_INSTANCE") ?? "my-instance";
+        public string DatabaseId = Environment.GetEnvironmentVariable("TEST_SPANNER_DATABASE") ?? $"my-db-{DateTimeOffset.UtcNow.ToUnixTimeSeconds()}";
         public string BackupDatabaseId = "my-test-database";
         public string BackupId = "my-test-database-backup";
-        public string ToBeCancelledBackupId = s_randomToBeCancelledBackupName;
-        public string RestoredDatabaseId = s_randomRestoredDatabaseName;
+        public string ToBeCancelledBackupId = $"my-backup-{DateTimeOffset.UtcNow.ToUnixTimeSeconds()}";
+        public string RestoredDatabaseId = $"my-restore-db-{DateTimeOffset.UtcNow.ToUnixTimeSeconds()}";
         public bool s_initializedDatabase { get; set; } = false;

```

Comment:
The naming is wrong here either

`public bool DatabaseInitialized {get; set;}` (do you need the set to be public?)
or `private static bool s_databaseInitialized`
or `private bool _databaseInitialized`

Reply:
This probably will remove in upcoming rewrite.